pipeline/special_sauce.rules

```diff
@@ -24,11 +24,13 @@ if "mhc_alleles" in config["input"] and _rna_exists():
     rule pyensembl:
       output:
         touch(join(dirname(config["reference"]["genome"]), "homo_sapiens_75.cached")),
+        touch(join(dirname(config["reference"]["genome"]), "homo_sapiens_93.cached")),
         touch(join(dirname(config["reference"]["genome"]), "mus_musculus_93.cached"))
       log:
         join(LOGDIR, "pyensembl_install.log")
       shell:
         "pyensembl install --release 75 --species 'homo sapiens' >> {log} 2>&1 && "
+        "pyensembl install --release 93 --species 'homo sapiens' >> {log} 2>&1 && "
```

Comment:
Eventually we'll probably want to release different docker containers for each reference genome, since we're currently limited only to stuff that gets installed here.